annotation/tests/test_distribution.py

```diff
@@ -1516,3 +1523,139 @@ def test_prepare_response():
             )
             == expected_output
         )
+
+
+@pytest.fixture
+def mock_session_and_tasks():
```

Comment:
Any block of code should do only one thing. Need to mock session? Session fixture. Need to mock tasks? Tasks fixture

https://realpython.com/solid-principles-python/